controller_manager/test/test_controller_manager_srvs.cpp

```diff
@@ -189,6 +190,71 @@ TEST_F(TestControllerManagerSrvs, list_controllers_srv)
   ASSERT_EQ(0u, result->controller.size());
 }
 
+TEST_F(TestControllerManagerSrvs, list_chained_controllers_srv)
+{
+  // create server client and request
+  rclcpp::executors::SingleThreadedExecutor srv_executor;
+  rclcpp::Node::SharedPtr srv_node = std::make_shared<rclcpp::Node>("srv_client");
+  srv_executor.add_node(srv_node);
+  rclcpp::Client<controller_manager_msgs::srv::ListControllers>::SharedPtr client =
```

Comment:
I am deeply disturbed by the bloat these fully-qualified names add to the test code.

Could you please set up a combination of `using ListControllers = controller_manager_msgs::srv::ListControllers;` and `using test_chainable_controller::TestChainableController` to eliminate namespacing from the test code?

That will convert this huge, hard-to-read block into something small and neat